### PR TITLE
Remove duplicate config of layer ch.bafu.schutzgebiete-aulav_uebrige

### PR DESCRIPTION
**Pull request by alcapat**
Not use the generalized table (uebrige_gebiete_general)

https://github.com/geoadmin/mf-chsdi3/issues/2460

### CODE REVIEW

**Comment by alcapat** [comment]
Rq : You can test this tooltip only with the topic aviation.

**Comment by gjn** [comment]
Works fine for me. Does this mean we can remove 1 table after this is merged and live?

**Comment by loicgasser** [comment]
Please also correct the test:

```
======================================================================
ERROR: test_htmlpopup_scale_dependent (chsdi.tests.integration.test_features_service.TestFeaturesView)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/var/lib/jenkins/workspace/PR_chsdi3/chsdi/tests/integration/test_features_service.py", line 216, in test_htmlpopup_scale_dependent
    resp.mustcontain('Les atterrissages en campagne')
  File "/var/lib/jenkins/workspace/PR_chsdi3/.venv/lib/python2.7/site-packages/webtest/response.py", line 366, in mustcontain
    "Body does not contain string %r" % s)
IndexError: Body does not contain string 'Les atterrissages en campagne'
-------------------- >> begin captured logging << --------------------
chsdi: DEBUG: route matched for url http://localhost/rest/services/all/MapServer/ch.bafu.schutzgebiete-aulav_uebrige/400/htmlPopup?lang=fr&imageDisplay=1362%2C1139%2C96&mapExtent=559349.7%2C127280.7%2C695549.7%2C241180.7; route_name: 'htmlPopup', path_info: u'/rest/services/all/MapServer/ch.bafu.schutzgebiete-aulav_uebrige/400/htmlPopup', pattern: '/rest/services/{map}/MapServer/{layerId}/{featureId}/htmlPopup', matchdict: {'featureId': u'400', 'map': u'all', 'layerId': u'ch.bafu.schutzgebiete-aulav_uebrige'}, predicates: ''
chsdi: DEBUG: debug_authorization of url http://localhost/rest/services/all/MapServer/ch.bafu.schutzgebiete-aulav_uebrige/400/htmlPopup?lang=fr&imageDisplay=1362%2C1139%2C96&mapExtent=559349.7%2C127280.7%2C695549.7%2C241180.7 (view name u'' against context <pyramid.traversal.DefaultRootFactory instance at 0x7f8487546170>): Allowed (no authorization policy in use)
--------------------- >> end captured logging << ---------------------
```

It should be nice to check the build logs when it fails.

File: chsdi/tests/integration/test_features_service.py [hunk 1]
```diff
@@ -210,14 +210,6 @@ def test_htmlpopup_lang(self):
         for msgid in msgids:
             self.assertIn(msgid, resp.text)
 
-    def test_htmlpopup_scale_dependent(self):
```
Comment (oterral) [inline chsdi/tests/integration/test_features_service.py]: Why removing the tests? `ch.bafu.schutzgebiete-aulav_uebrige` still exist.

Comment (alcapat) [inline chsdi/tests/integration/test_features_service.py]: Yes, ch.bafu.schutzgebiete-aulav_uebrige still exist but without scale dependencie

**Comment by loicgasser** [comment]
Thanks!

**Pull request merged**